ISCSI/SCSI/Enums/ModePageCodeName.cs

```diff
@@ -8,6 +8,7 @@ public enum ModePageCodeName : byte
         ControlModePage = 0x0A,
         PowerConditionModePage = 0x1A,
         InformationalExceptionsControlModePage = 0x1C,
+        MMCapabilitiesAndMechanicalStatus = 0x2A,

```

Comment:
Isn't this device specific? which specification covers that 0x2A code?

Reply:
This is CD/DVD device specific (MMC), Windows requests it, for example, when peripheral device type is 0x5.
You can find it's description in scsi spec paragraph 5.2.3.4 (http://www.13thmonkey.org/documentation/SCSI/x3_304_1997.pdf)